platforms/intellij/compose/playground/src/jvmMain/kotlin/foundry/intellij/compose/playground/MarkdownPlayground.kt

```diff
@@ -31,11 +33,20 @@ import org.jetbrains.jewel.intui.standalone.theme.IntUiTheme
 import org.jetbrains.jewel.ui.component.DefaultButton
 import org.jetbrains.jewel.ui.component.Text
 
-fun main() = singleWindowApplication {
+fun main() {
+  singleWindowApplication { AppContent() }
+}
+
+// created function for testing
```

Comment:
I added a comment here for now since testing on the function wrapped in a singleWindowApplication seemed to result in an infinite loop (where the test wouldn't end). I made a composable `AppContent()` function with the same content and added a test tag too.

Reply:
I don't think we need the comment, feels obvious in source why it's there :)